### ISSUE TITLE
Missing hover transitions

### ISSUE THREAD

**Issue by flipflop97** [flathub/website#1258]
All buttons in the content body, as well as app cards and many other elements, have a hover transition. Some elements don't have one however. Giving these elements an equivalent hover transition increases consistency across the site. The elements I noticed that are missing one are:

- [ ] Headerbar buttons
- [ ] Account menu buttons
- [ ] Account page tabs
- [ ] Copy button of manual app install commands

**Comment by razzeee** [comment]
Not sure I follow, can you screenshot the components? The one I thought you mean all have hover states. Or do you mean the animation is instant and has no fadein/transition?

**Comment by flipflop97** [comment]
Yes, I meant the hover transition/animation/fade-in. They all do have hover states, just no transitions.
